plugins/services/src/js/structs/Framework.js

```diff
@@ -8,13 +8,18 @@ import Application from "./Application";
 import FrameworkSpec from "./FrameworkSpec";
 
 module.exports = class Framework extends Application {
-  getName() {
+  getPackageName() {
+    const labels = this.getLabels();
+    if (labels && labels.DCOS_PACKAGE_NAME) {
+      return labels.DCOS_PACKAGE_NAME;
+    }
```

Comment:
The `getLabels()` method will already ensure an object is returned so this really only needs to be
`return this.getLabels()['DCOS_PACKAGE_NAME'];`